### ISSUE TITLE
bump third-party/libdeflate when next version is released

### ISSUE THREAD

**Issue by ckerr** [transmission/transmission#3831]
libdeflate migrated to CMake in https://github.com/ebiggers/libdeflate/pull/235, which means Transmission could drop its cmake patch and just use upstream directly. We should bump `electron/libdeflate` after upstream's next release.

**Comment by GaryElshaw** [comment]
Just a note to say that change has landed, if you hadn't seen, and it compiles fine.

**Comment by ckerr** [comment]
> Just a note to say that change has landed, if you hadn't seen, and it compiles fine.

Yep, I commented on it at https://github.com/ebiggers/libdeflate/pull/101#issuecomment-1263778867
